app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java

```diff
@@ -159,6 +163,18 @@ private void buyTicket() {
         startActivity(intent);
     }
 
+    private void addToCalendar() {
+        Intent intent = new Intent(Intent.ACTION_INSERT)
+                .setData(CalendarContract.Events.CONTENT_URI)
+                .putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, event.getDate().getTime())
+                .putExtra(CalendarContract.EXTRA_EVENT_END_TIME, event.getDate().getTime() + 7200000)
+                .putExtra(CalendarContract.Events.TITLE, event.getTitle())
+                .putExtra(CalendarContract.Events.DESCRIPTION, event.getDescription())
+                .putExtra(CalendarContract.Events.EVENT_LOCATION, event.getLocality())
+                .putExtra(CalendarContract.Events.AVAILABILITY, CalendarContract.Events.AVAILABILITY_BUSY);
```

Comment:
What does this line do? 
This might be a good place to add a comment to explain what it does and why we need it as I  do not understand it from just reading the code.

Reply:
I change it to CalendarContract.Events.AVAILABILITY_FREE. It indicates that this event is free time and will not conflict with other events. I make a comment here.